apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -3,23 +3,24 @@ import { Animated, TouchableOpacity, Text, StyleSheet, View, FlatList } from 're
 import FastImage from 'react-native-fast-image'
 import Intercom from '@intercom/intercom-react-native'
 import { Globe, Plus, CircleHelp } from 'lucide-react-native'
-import { clsx } from 'clsx'
 import { map, sortBy } from 'lodash/fp'
+import { clsx } from 'clsx'
+import { openURL } from 'hooks/useOpenURL'
 import useCurrentUser from '@hylo/hooks/useCurrentUser'
-import useCurrentGroup from '@hylo/hooks/useCurrentGroup'
-import { PUBLIC_GROUP } from '@hylo/presenters/GroupPresenter'
+import useCurrentGroup, { useContextGroups } from '@hylo/hooks/useCurrentGroup'
 import useChangeToGroup from 'hooks/useChangeToGroup'
-import { openURL } from 'hooks/useOpenURL'
 
 export default function ContextSwitchMenu () {
+  const changeToGroup = useChangeToGroup()
   const [{ currentUser }] = useCurrentUser()
   const [{ currentGroup }] = useCurrentGroup()
-  const changeToGroup = useChangeToGroup()
-  const myGroups = [PUBLIC_GROUP].concat(sortBy('name', map(m => m.group, currentUser.memberships)))
+  const { myContext, publicContext } = useContextGroups()
+  const myGroups = [myContext, publicContext].concat(
```

Comment:
Wait, are you putting another way to access the my context here?

Reply:
yes see GroupPresenter

Reply:
no, i mainly meant as a piece of the UI, which you have clarified elsewhere.